WordPress/src/main/java/org/wordpress/android/ui/main/jetpack/migration/JetpackMigrationViewModel.kt

```diff
@@ -5,42 +5,85 @@ import dagger.hilt.android.lifecycle.HiltViewModel
 import kotlinx.coroutines.flow.MutableStateFlow
 import kotlinx.coroutines.flow.StateFlow
 import org.wordpress.android.R
+import org.wordpress.android.fluxc.store.AccountStore
+import org.wordpress.android.ui.main.jetpack.migration.JetpackMigrationViewModel.ActionButton.WelcomePrimaryButton
+import org.wordpress.android.ui.main.jetpack.migration.JetpackMigrationViewModel.ActionButton.WelcomeSecondaryButton
 import org.wordpress.android.ui.main.jetpack.migration.JetpackMigrationViewModel.StepUiState.Welcome
+import org.wordpress.android.ui.utils.UiString
+import org.wordpress.android.ui.utils.UiString.UiStringRes
+import org.wordpress.android.util.GravatarUtilsWrapper
 import javax.inject.Inject
 
+const val USER_AVATAR_SIZE = 32 * 3
```

Comment:
This is more of a question than a suggestion for a change. Does the avatar size have to do with the icon size multiplied with the density? In that case should we fetch the density dynamically?

Reply:
> This is more of a question than a suggestion for a change. Does the avatar size have to do with the icon size multiplied with the density? In that case should we fetch the density dynamically?

Thank you for raising this point @antonis 🙇 

It's correct, that is the better way to calculate the avatar size, since it should indeed be a product of the icon size and screen density.

I'll keep this in mind for future improvements and provide a solution in the next PR for this screen, which should implement the design update.